Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Collections/GridObjectCollection.cs

```diff
@@ -0,0 +1,482 @@
+                    for (int i = 0; i < NodeList.Count; i++)
+                    {
+                        newPos = ScatterMapping(nodeGrid[i], Radius);
+                        Collider nodeCollider = NodeList[i].transform.GetComponentInChildren<Collider>();
+                        if (nodeCollider != null)
+                        {
+                            // Make the radius the largest of the object's dimensions to avoid overlap
+                            Bounds bounds = nodeCollider.bounds;
+                            NodeList[i].Radius = Mathf.Max(Mathf.Max(bounds.size.x, bounds.size.y), bounds.size.z) / 2;
+                        }
+                        else
+                        {
+                            NodeList[i].Radius = Radius;
+                        }
+                        NodeList[i].transform.localPosition = newPos;
+                        switch (OrientType)
+                        {
+                            case OrientationType.FaceOrigin:
+                            case OrientationType.FaceFoward:
+                                NodeList[i].transform.rotation = Quaternion.LookRotation(newRot);
+                                break;
+
+                            case OrientationType.FaceOriginReversed:
+                            case OrientationType.FaceForwardReversed:
+                                NodeList[i].transform.rotation = Quaternion.LookRotation(newRot);
+                                NodeList[i].transform.Rotate(0f, 180f, 0f);
+                                break;
+
+                            case OrientationType.None:
+                                break;
+
+                            default:
+                                throw new ArgumentOutOfRangeException();
+                        }
+                    }
+
+                    // Iterate [x] times
+                    for (int i = 0; i < 100; i++)
+                    {
+                        IterateScatterPacking(NodeList, Radius);
+                    }
+                    break;
+            }
+        }
+
+        // Internal function for getting the relative mapping based on a source Vec3 and a radius for spherical mapping.
+        // The source to be mapped to sphere, radius the radius of the sphere</param>
+        private Vector3 SphericalMapping(Vector3 source, float radius)
+        {
+            Radius = radius >= 0 ? Radius : radius;
+            Vector3 newPos = new Vector3(0f, 0f, Radius);
+
+            float xAngle = (source.x / circumferenc) * 360f;
+            float yAngle = -(source.y / circumferenc) * 360f;
+
+            Quaternion rot = Quaternion.Euler(yAngle, xAngle, 0.0f);
+            newPos = rot * newPos;
+
+            return newPos;
+        }
+
+        // Internal function for getting the relative mapping based on a source Vec3 and a radius for cylinder mapping.
+        // source to be mapped to cylinder, Radius for the radius of the cylinder
```

Comment:
Should these be xml format?

```
/// <summary>
/// Comments
/// </summary>
```

Reply:
@davidkline-ms marked a change to drop them since they're private/internal. I figured I'd leave the explanation - just removed the XML format.

Reply:
I was just looking for "Internal function for getting the ..." to be changed to "Get the ..."

There is no real need to specify the visibility in the comment (it's easy to see in the method signature :)